layouts/index.html

```diff
@@ -132,7 +132,7 @@ <h3>
       <li><a href="/docs/overview/">About</a></li>
       <li><a href="/gsoc/">GSoC</a></li>
     </ul>
-    <p>Built with all the love in the world by <a href="https://twitter.com/carllerche" target="_blank" rel="noopener">@carllerche</a> with the help of <a href="https://github.com/tokio-rs/tokio/graphs/contributors">our contributors</a>.</p>
+    <p>Built with all the love in the world by <a href="https://twitter.com/carllerche" target="_blank" rel="noopener">@carllerche</a> with the help of <a href="https://github.com/tokio-rs/tokio/graphs/contributors">our contributors</a>. Deployed with <a href="https://netlify.com>Netlify</a>.</p>
```

Comment:
is this site actually deployed with netlify or is it just tracing?

Reply:
Just `https://tracing.rs` atm, but @carllerche mentioned that he'd like to get https://tokio.rs on Netlify as well.